app/controllers/api_controller/container_deployments.rb

```diff
@@ -0,0 +1,12 @@
+class ApiController
+  module ContainerDeployments
+    def start_resource_container_deployments(_type, _id, data)
```

Comment:
This is a method that will be avaviliable for the rest-api, exmaple:
{
"action": "start",
"resource" : {
  "providerName":"dfssasaasdsasada",
  "providerType":"openshiftEnterprise",
  ....
}